app/src/main/java/de/danoeh/antennapod/fragment/EpisodesListFragment.java

```diff
@@ -174,7 +174,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         recyclerView.setRecycledViewPool(((MainActivity) getActivity()).getRecycledViewPool());
         setupLoadMoreScrollListener();
 
-        swipeActions = new SwipeActions(this, getFragmentTag()).attachTo(recyclerView);
+        swipeActions = new SwipeActions(this, TAG).attachTo(recyclerView);
```

Comment:
The old code was correct here. Child classes of EpisodesListFragment are used in playback history, all episodes list, and inbox. Using the method call ensures that we store the tag of the respective child fragment.

Reply:
OK, thanks for the helpful clarification, I'll get this fixed up.

Reply:
How do I properly reference `getFragmentTag()` in SwipePreferencesFragment?

Reply:
I've reverted the code that should not have been modified, but the code still doesn't quite work. If you go to the Episodes list after updating, and try to swipe, it launches the swipe preferences dialog and forces you to set the swipe action again. Once set, the new Settings menu preference matches what you set in the dialog, but the original value is lost. Ideally, we would preserve whatever actions were set prior to the upgrade.

Reply:
> I think I figured it out, trying using AllEpisodesFragment.TAG there.

Yeah, exactly. That was the idea :)

> If you go to the Episodes list after updating, and try to swipe, it launches the swipe preferences dialog and forces you to set the swipe action again.

That's weird. I would not expect it to do that. Isn't the tag used for saving the same now? Note that it does ask on the very first swipe. Maybe your upgrade code removed the setting? Could you maybe try again, switching between `develop` and your PR?

Reply:
> Could you maybe try again, switching between develop and your PR?

Yeah, this is how I was testing. I think I must have forgotten to download the new APK from my laptop and I was testing with an earlier build, because I just downloaded it again and it works as expected.

It should be good to merge.